### PR TITLE
Re-pin EF dependencies

### PR DESCRIPTION
**Pull request by dougbu**
- not shipping this time around
- also bump a few versions to match newer 'release/2.1' dependencies
  - System.Diagnostics.DiagnosticSource
  - System.Security.Principal.Windows
  - System.Threading.Tasks.Extensions

### CODE REVIEW

**Comment by mmitche** [comment]
I think you mean "Pin ef dependencies"

**Review by mmitche — APPROVED** [review]

**Comment by dougbu** [comment]
Customers not using standalone apps will get the latest from the shared framework i.e. roll-forward will help for the most part. Does that mean we don't need to rebuild Microsoft.AspNetCore.Server.HttpSys?

W.r.t . SharedFx: Assemblies for the 3 updated packages have file and product version 4.6.27818.2 in aspnetcore-runtime-2.2.6-win-x64.zip
- seems newer than the 4.6.27818.1 versions in aspnetcore-runtime-2.1.12-win-x64.zip but it's hard to be sure

**Review by ericstj — APPROVED** [review]

File: build/dependencies.props [hunk 1]
```diff
@@ -231,18 +228,18 @@
...
     <SystemSecurityCryptographyXmlPackageVersion>4.5.0</SystemSecurityCryptographyXmlPackageVersion>
     <SystemSecurityPermissionsPackageVersion>4.5.0</SystemSecurityPermissionsPackageVersion>
-    <SystemSecurityPrincipalWindowsPackageVersion>4.5.0</SystemSecurityPrincipalWindowsPackageVersion>
+    <SystemSecurityPrincipalWindowsPackageVersion>4.5.1</SystemSecurityPrincipalWindowsPackageVersion>
     <SystemServiceProcessServiceControllerPackageVersion>4.5.0</SystemServiceProcessServiceControllerPackageVersion>
     <SystemTextEncodingsWebPackageVersion>4.5.0</SystemTextEncodingsWebPackageVersion>
     <SystemThreadingChannelsPackageVersion>4.5.0</SystemThreadingChannelsPackageVersion>
     <SystemThreadingTasksDataflowPackageVersion>4.9.0</SystemThreadingTasksDataflowPackageVersion>
-    <SystemThreadingTasksExtensionsPackageVersion>4.5.1</SystemThreadingTasksExtensionsPackageVersion>
+    <SystemThreadingTasksExtensionsPackageVersion>4.5.2</SystemThreadingTasksExtensionsPackageVersion>
```
Comment (ericstj) [inline build/dependencies.props]: I believe the latest version is actually 4.5.3

**Review by ericstj — COMMENTED** [review]
LGTM if you fix S.Threading.Tasks.Extensions

File: build/dependencies.props (hunk 1 above)
Comment (dougbu) [reply build/dependencies.props]: Should we be using the latest in 'release/2.1' as well?

Comment (ericstj) [reply build/dependencies.props]: Probably, but I wouldn't spin a new build just to ship that fix.  Maybe make the change and then ship it the next time you update packages which depend on it.

**Pull request merged**